rpm/SPECS/docker-scan-plugin.spec

```diff
@@ -12,7 +12,7 @@ URL: https://github.com/docker/scan-cli-plugin/
 Vendor: Docker
 Packager: Docker <support@docker.com>
 
-Requires: docker-ce-cli
+Enhances: docker-ce-cli
```

Comment:
Only thinking if this should be set conditional as well (not sure if older versions just ignore it, or if they would warn/complain), i.e.;

```suggestion
# CentOS 7 and RHEL 7 do not yet support weak dependencies
 %if 0%{?rhel} != 7
 Enhances: docker-ce-cli
 %endif
```

Reply:
ok; looks like this is needed

```
error: line 15: Unknown tag: Enhances: docker-ce-cli
```